integration-tests/test_ros.py

```diff
@@ -0,0 +1,187 @@
+"""
+:component: insights-client-ros
+:requirement: RHSS-291297
+:polarion-project-id: RHELSS
+:polarion-include-skipped: false
+:polarion-lookup-method: id
+:poolteam: rhel-sst-csi-client-tools
+:caseautomation: Automated
+:upstream: Yes
+"""
+
+import pytest
+import subprocess
+import conftest
+import shutil
+from constants import CONFIG_FILE
+from pathlib import Path
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+PACKAGE = "insights-client-ros"
+SERVICE = "pmlogger"
+
+
+@pytest.mark.tier1
+def test_ros_install():
+    """
+    :id: 3fc19957-be97-429d-817a-610e6c69dd9d
+    :title: Verify insights-client-ros can be installed
+    :description:
+        Ensure that the insights-client-ros can be successfully installed
+    :tags: Tier 1
+    :steps:
+        1. Try to install insights-client-ros
+        2. Check that ros_collect is set to true in insights-client.conf
+    :expectedresults:
+        1. Subpackage insights-client-ros is installed.
+        2. ros_collect is set to true in insights-client.conf
```

Comment:
Just a nitpick, to be consistent, remove the ending dot in the 1st expected results. Also `ros_collect` should start with a capital letter according to [your comment](https://github.com/RedHatInsights/insights-client/pull/437#discussion_r2145086104):
```suggestion
        1. Subpackage insights-client-ros is installed
        2. ros_collect is set to true in insights-client.conf
```